README.md

```diff
@@ -23,4 +23,7 @@ Currently built on Moltin but PRs for other systems are welcome
 - rename the `/doc/.env.template` to `/.env` and fill out the details
 
 ## More information
-- check the [Philosophy document](https://github.com/KickAssCommerce/KickAssCommerce/blob/develop/doc/philosophy.md)
\ No newline at end of file
+- check the [Philosophy document](https://github.com/KickAssCommerce/KickAssCommerce/blob/develop/doc/philosophy.md)
+
+## Examples
+Anything in the `app` directory is example code. The `app` directory should be used for your project code.
```

Comment:
Oh I like this! Maybe we should consider in future having a sample demo repo that people can look at that builds an app directory with "working" examples. Keeping this repo clean while still letting people see how to use our a-m-a-z-i-n-g tool.